.gitmodules

```diff
@@ -89,7 +89,7 @@
 	url = https://github.com/conp-bot/conp-dataset-Comparing-Perturbation-Modes-for-Evaluating-Instabilities-in-Neuroimaging-Processed-NK.git
 [submodule "projects/multicenter-phantom"]
 	path = projects/multicenter-phantom
-	url = https://github.com/emmetaobrien/multicenter-phantom.git
+	url = https://github.com/conpdatasets/multicenter-phantom.git
```

Comment:
@emmetaobrien Nice! You were able to resolve the git-annex branch conflicts in the end with conpdatasets?

Reply:
I was able to resolve those conflicts, yes.  I am correcting the `mousebytes` field now.